libs/service-portal/settings/access-control/src/screens/GrantAccess/GrantAccess.tsx

```diff
@@ -69,12 +70,22 @@ function GrantAccess() {
   }, [authDelegation, setValue])
 
   const onSubmit = handleSubmit(async ({ toNationalId, name }) => {
-    const { data, errors } = await createAuthDelegation({
```

Comment:
The error link will throw an error so errors will always be undefined here, thus handled by try catch.